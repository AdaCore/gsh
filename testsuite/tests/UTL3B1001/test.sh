basename
basename a b c
basename ""
basename "."
basename /
basename \\
basename //
basename \\\\
basename /toto\\tata.txt a.txt
