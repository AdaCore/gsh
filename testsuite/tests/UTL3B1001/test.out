[error] basename: need at least one operand
[error] basename: does not accept more than two operands

.
/
/
//
\\
tat
